pkg/csi/cinder/controllerserver.go

```diff
@@ -266,7 +267,7 @@ func (cs *controllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateS
 		return nil, errors.New("multiple snapshots reported by Cinder with same name")
 	} else {
 		// TODO: Delegate the check to openstack itself and ignore the conflict
-		snap, err = cs.Cloud.CreateSnapshot(name, volumeId, description, &req.Parameters)
+		snap, err = cs.Cloud.CreateSnapshot(name, volumeId, &properties)
```

Comment:
IMO we should not remove req.parameters as it can be used in future

Reply:
ok